.copr/Makefile

```diff
@@ -0,0 +1,38 @@
+
+
+SELF=$(lastword $(MAKEFILE_LIST))
+ROOT_DIR=$(abspath $(dir $(SELF))/..)
+SKIP_DEPS=
+
+outdir:=/var/tmp/copr-tmp-outdir
+spec:=extras/python-sambacc.spec
+
+.PHONY: srpm
+srpm: sys_deps py_deps
+	mkdir -p $(outdir)
+	git fetch --tags
+	SAMBACC_SRPM_ONLY=yes \
+		SAMBACC_BUILD_DIR=$(ROOT_DIR) \
+		SAMBACC_DIST_PREFIX=$(outdir)/.dist \
+		SAMBACC_DISTNAME=copr \
+		SAMBACC_BUILD_TASKS="task_py_build task_rpm_build" \
+		 ./tests/container/build.sh
+	cp $(outdir)/.dist/copr/SRPMS/*.rpm  $(outdir)
+
+
+.PHONY: sys_deps
+sys_deps:
+ifeq ($(SKIP_DEPS),yes)
+	@echo "Skipping sys deps"
+else
+	dnf install -y python3-pip git
+endif
+
+
+.PHONY: py_deps
+py_deps:
+ifeq ($(SKIP_DEPS),yes)
+	@echo "Skipping py deps"
+else
+	pip install build
+endif
```

Comment:
I suppose we could. I was really torn between just duplicating the logic from build.sh, refactoring build.sh, rewriting build.sh, or doing what I ended up which I considered to be minimally hacking it to make portions reusable outside the container.
I ended up putting these rules in the makefile because this was the minimal set of things I needed to do to get the build working in the COPR environment.